packages/gatsby/src/redux/actions/restricted.js

```diff
@@ -0,0 +1,276 @@
+// @flow
+const _ = require(`lodash`)
+const report = require(`gatsby-cli/lib/reporter`)
+
+import type { Plugin } from "./types"
+
+const actions = {}
+
+/**
+ * Add a third-party schema to be merged into main schema. Schema has to be a
+ * graphql-js GraphQLSchema object.
+ *
+ * Only available in [`sourceNodes`](/docs/node-apis/#sourceNodes) API.
```

Comment:
This is ok, but for future maybe we could use some custom jsdoc tag to mark in what APIs action is available, so we can display it nicely in our action reference docs (so it isn't lost in description's wall of text)

Reply:
will `documentationjs` parse custom tags?

Reply:
It does parse in show just full bag of all tags (even custom ones), we probably would need to create extra field in `DocumentationJS` type with something like `createResolvers` to access it nicely for .org (as this would be custom thing not applicable for all users of the plugin)

Reply:
but this is not a blocker for this PR - just sidenote

Reply:
Two questions:
* `gatsby-transformer-documentationjs` needs to preserve custom tags on the created nodes. better to preserve all, or just the one we care about (`@availableIn`)? I've opted for the latter now.
* we would not need to use `createResolvers` if we rely on inference - I've however now added createResolvers call and made the `DocumentationJs` type `@dontInfer`